src/main/java/sc/iview/SciView.java

```diff
@@ -2161,7 +2161,12 @@ public String nodeInfoString(Node n) {
     /**
      * Static launching method
      */
+    @Deprecated
```

Comment:
Whenever I put `@Deprecated` on something, I try to always also put `@deprecated Use {@link Foo#bar()} instead.` in the accompanying javadoc. Otherwise it can be frustrating to hit a "deprecation dead end" without guidance on how to avoid using the deprecated features.

Reply:
Ah, thanks!